docs/02-app/02-api-reference/02-file-conventions/layout.mdx

```diff
@@ -162,8 +181,71 @@ Common `pathname` patterns can also be implemented with [`params`](#params-optio
 
 See the [examples](/docs/app/building-your-application/routing/layouts-and-templates#examples) section for more information.
 
+## Examples
+
+### Displaying content based on `params`
+
+Using [dynamic route segments](/docs/app/building-your-application/routing/dynamic-routes), you can display or fetch specific content based on the `params` prop.
+
+```tsx filename="app/dashboard/layout.tsx" switcher
+export default async function DashboardLayout({
+  children,
+  params,
+}: {
+  children: React.ReactNode
+  params: Promise<{ team: string }>
+}) {
+  const { team } = await params
+
+  return (
+    <section>
+      <header>
+        <h1>Welcome to {team}'s Dashboard</h1>
+      </header>
+      <main>{children}</main>
+    </section>
+  )
+}
+```
+
+```jsx filename="app/dashboard/layout.js" switcher
+export default async function DashboardLayout({ children, params }) {
+  const { team } = await params
+
+  return (
+    <section>
+      <header>
+        <h1>Welcome to {team}'s Dashboard</h1>
+      </header>
+      <main>{children}</main>
+    </section>
+  )
+}
+```
+
+### Reading `params` in synchronous components
+
+To use `params` in a synchronous Server Component, you can use React's [`use`](https://react.dev/reference/react/use) function to read the promise:
+
+```tsx filename="app/page.tsx" switcher
+import { use } from 'react'
+
+export function Page({ params }: { params: Promise<{ slug: string }> }) {
+  const { slug } = use(params)
+}
```

Comment:
Is it pedantic to suggest that this code block show a full page return alongside the usage of `use` ?

Reply:
It was omitted because it's not relevant to the example, we could just return an empty tag but seems redundant since the main example includes a return.